no_drama/build_lib/dfd_paths.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+
+this_file = os.path.realpath(__file__)
+this_dir = os.path.dirname(this_file)
+root = os.path.realpath(os.path.join(this_dir, '../'))
+
+path_file_path = os.path.join(root, 'paths.json')
+
+paths = {'environment': 'environment.json',
+         'static_in': '../../static.in',
+         'extended_python_path': '../../lib',
+         'static_out': 'static',
+         'update_symlink': '../../current',
+         'extended_wsgi':'extended.wsgi',
+         }
+
+if os.path.exists(path_file_path):
+    with open(path_file_path) as path_file:
+        configured_paths = json.load(path_file)
+        paths.update(configured_paths)
+
+
+def abs_path(name):
+    relpath = paths[name]
+    if relpath is not None and bool(relpath) is True:
+        joined = os.path.join(root, relpath)
+        return os.path.normpath(joined)
+    else:
+        return None
```

Comment:
I think Robert Martin talks about this with nulls in java.  The problem is it is unclear what none signals.  Is it an error?  Does it just mean that it can be safely ignored?  Different programmers use none and null in different contexts to mean different things causing ambiguity.

This is one reason you'll see the error none type has no attribute...or a null pointer exception in java.